browser/resources/settings/brave_overrides/settings_section.ts

```diff
@@ -11,11 +11,25 @@ RegisterStyleOverride(
   html`
     <style include="settings-shared">
       #header .title {
-        font-weight: 500 !important;
+        font-weight: 600 !important;
+        font-size: var(--leo-typography-heading-h3-font-size) !important;
```

Comment:
I take it we can't just use:

```suggestion
        font: var(--leo-heading-h3) !important;
```

for some reason?

Reply:
Ohhhh yeah we could, forgot about that